docs/content/en/docs/pipeline-stages/cleanup.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Cleanup"
+linkTitle: "Cleanup"
+weight: 60
+featureId: cleanup
+---
+
+Skaffold works with [image builders]({{<relref "/docs/pipeline-stages/builders">}}) and [deployers]({{<relref "/docs/pipeline-stages/deployers">}}) 
+that both have side effects on the environment: 
+
+- resources are created in one or more namespaces in a Kubernetes cluster 
+- images might be created on the local Docker daemon
+- images might be pushed to registries
+- application might have arbitrary side effects 
```

Comment:
```suggestion
- deployed applications might have unintended side effects 
```

Reply:
I don't agree with this - applications typically have very much intended side effects, like new records in databases, messages sent, etc.